src/python/pants/testutil/test_base.py

```diff
@@ -399,13 +399,14 @@ def assertDoesNotRaise(self, exc_class: Type[BaseException] = Exception):
         except exc_class as e:
             raise AssertionError(f"section should not have raised, but did: {e}") from e
 
-    def get_bootstrap_options(self, cli_options=()):
+    @staticmethod
+    def get_bootstrap_options(cli_options=()):
         """Retrieves bootstrap options.
 
         :param cli_options: An iterable of CLI flags to pass as arguments to `OptionsBootstrapper`.
         """
         args = tuple(["--pants-config-files=[]"]) + tuple(cli_options)
-        return OptionsBootstrapper.create(args=args).bootstrap_options.for_global_scope()
+        return OptionsBootstrapper.create(env={}, args=args).bootstrap_options.for_global_scope()
```

Comment:
Wow. Were all of these parsing the pytest environment before? Sheesh.

Reply:
Yep!